deliver/lib/deliver/app_screenshot.rb

```diff
@@ -136,6 +138,7 @@ def formatted_name
         ScreenSize::IOS_IPAD_10_5 => "iPad 10.5",
         ScreenSize::IOS_IPAD_11 => "iPad 11",
         ScreenSize::IOS_IPAD_PRO => "iPad Pro",
+        ScreenSize::IOS_IPAD_PRO129_3RD => "iPad Pro (12.9-inch) (3rd generation)",
```

Comment:
`ScreenSize::IOS_IPAD_PRO => "iPad Pro",` above is second generation, correct?
If so, wouldn't it make more sense to leave the `_3RD`?

Reply:
Nope, the naming is adhoc here.
I try to follow Xcode's simulator naming:
<img width="310" alt="Screenshot 2019-04-13 at 18 21 47" src="https://user-images.githubusercontent.com/3721061/56081726-2174b200-5e19-11e9-87cc-e22e6bb997f5.png">
I guess `Pro` here is quite redundant so I suggest renaming (taking into account https://help.apple.com/app-store-connect/#/devd274dd925):
`ScreenSize::IOS_IPAD_PRO` to `ScreenSize::IOS_IPAD_12_9_2ND` and
`ScreenSize::IOS_IPAD_PRO129_3RD` to `ScreenSize::IOS_IPAD_12_9_3RD`
Would it be ok?

Reply:
No, these _are_ an iPad Pros - which are a different family of devices.

I guess `IOS_IPAD_PRO_12_9` could be defined as being the 3rd gen (vs. `IOS_IPAD_PRO` being 1st and 2nd gen). Inconsistent, but in a consistent way ;)